test/unit/mwApi.test.ts

```diff
@@ -40,7 +40,7 @@ describe('mwApi', () => {
     expect(United_Kingdom).toBeDefined()
 
     // Article "United_Kingdom" has categories
-    expect(United_Kingdom?.categories?.length).toBeGreaterThanOrEqual(12)
+    expect(United_Kingdom?.categories?.length).toBeGreaterThanOrEqual(1)
```

Comment:
Why `1` and not the exact number of categories currently available in the WPEN London article? `1` is not really meaningful.

Reply:
Fixed